### PR TITLE
Typing the table store

### PR DESCRIPTION
**Pull request by adrinr**
## Description
Type table store

### CODE REVIEW

File: packages/builder/src/stores/builder/tables.ts [hunk 1]
```diff
@@ -99,28 +133,33 @@ export function createTablesStore() {
...
+      await this.fetch()
     } else {
-      await Promise.all(tableIdsToFetch.map(id => singleFetch(id)))
+      await Promise.all(tableIdsToFetch.map(id => this.singleFetch(id)))
     }
     return savedTable
   }
 
-  const deleteTable = async table => {
+  async delete(table: { _id: string; _rev: any }) {
```
Comment (mike12345567) [inline packages/builder/src/stores/builder/tables.ts:143]: Why does `_rev` need to be any here - is `_rev?: string` not sufficient?

File: packages/builder/src/stores/builder/tables.ts [hunk 2]
```diff
@@ -198,26 +237,12 @@ export function createTablesStore() {
     }
   }
 
-  const removeDatasourceTables = datasourceId => {
-    store.update(state => ({
+  removeDatasourceTables(datasourceId: any) {
```
Comment (mike12345567) [inline packages/builder/src/stores/builder/tables.ts:240]: can datasource ID not be specified as a string?

File: packages/builder/src/stores/builder/tables.ts (hunk 1 above)
Comment (adrinr) [reply packages/builder/src/stores/builder/tables.ts]: Yup, mistake. Done. It should always be required, and it looks like there is a single usage were we are passing the full table

**Review by mike12345567 — APPROVED** [review]
Nice job!

**Pull request merged**